pkg/autoops/api/progressive_rollout.go

```diff
@@ -42,8 +42,8 @@ const (
 )
 
 var (
-	errProgressiveRolloutAutoOpsWebhookClauseExists  = errors.New("progressive rollout: ")
-	errProgressiveRolloutAutoOpsDatetimeClauseExists = errors.New("")
+	errProgressiveRolloutAutoOpsWebhookClauseExists  = errors.New("progressive rollout: can not use progressive rollout when auto ops date time clause exists")
```

Comment:
nit: `autoops: can not create a progressive rollout when the schedule is set in the auto ops`